boot_default/config.txt

```diff
@@ -75,35 +76,57 @@ display_rotate=0
 # Decrease audio noise.
 disable_audio_dither=1
 
+# ---------------------------------------------------------------------------------------
+# Configuration Filters - RPis
 
-# for more options see http://elinux.org/RPi_config.txt
+[pi2]
+# Give more current to the USB ports for the Pixel Kit.
+max_usb_current=1
+[all]
 
-# Configuration Filters
+[pi3]
+# for light board
```

Comment:
I wasn't aware we don't force max_usb_current on the PI3. We are sure we don't need it for the USB hub, right?

Reply:
The `max_usb_current` option is only for Pi2, for Pi3s the USBs already push out the max current.